persist-cli/src/main/java/io/ballerina/persist/nodegenerator/BalSyntaxConstants.java

```diff
@@ -235,5 +237,6 @@ private InheritedTypeReferenceConstants() {
 
 
     public static final String MYSQL_DRIVER = "mysql.driver";
+    public static final String FILE_PATH_SEPARATOR = Pattern.quote(System.getProperty("file.separator"));
```

Comment:
Do we need this constant.

Reply:
removed in [fb621ad](https://github.com/ballerina-platform/persist-tools/pull/83/commits/fb621ad4f4d1fb538bf618cef7dfaf37436c5292)